PublicMapping/districtbuilder
# Add organization details

## Overview

Adds new fields to `Organization` entity, which are used to populate the organization details screen.

Also adds a new command to add or update an organization from a YAML configuration file

Along the way I also fixed an issue with our migrations - after updating TypeORM it kepy trying to recreate our constraints in order to change their naming scheme, and additionally we needed to change the way we specify our default for the `locked_districts` column (https://github.com/PublicMapping/districtbuilder/commit/7cb733eb094c1e633ad5ad644729e685899d0573).

### Checklist


### Demo

With a logo set:
![localhost_3003_o_mikes-maps_ (3)](https://user-images.githubusercontent.com/4432106/106916379-1917f600-66d5-11eb-9bc4-41aa8566eeee.png)
Without a logo:
![localhost_3003_o_mikes-maps_ (2)](https://user-images.githubusercontent.com/4432106/106916381-1917f600-66d5-11eb-81d6-5207e33609d7.png)


### Notes

I used this YAML (ipsum text provided by [Hodor ipsum](https://ipsum-community.github.io/hodoripsum/):
```
name: Mike's Maps
slug: mikes-maps
description:
  HODOR HODOR! Hodor! Hodor hodor. Hodor? HODOR? Hodor, hodor, hodor hodor. Hodor? Hodor, hodor. Hodor. Hodor. Hodor! Hodor hodor.
  Hodor, hodor, hodor hodor. Hodor hodor HODOR! Hodor. Hodor! Hodor hodor. HODOR? HODOR hodor, hodor. Hodor? Hodor, hodor. Hodor!
logoUrl: data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAEoAAABLCAYAAADXjBHUAAACr0lEQVR4Ae3aAWQbURwG8FtbqgUAMBgAGIAB7F6zVVc6FDDAvQa7pGGAzcIMGFarLS/AGORsAENeqsDAwDCmBbUNStzLJrPB252l26GWL7KT/eP7+AtBJL98Hv7vAkZgam17J251+7WW7TTN21WKnIdkevcyJF+Yg/FYRCIWjkSsSZCIVW/ZuyASjkUkHItIOBaRcCwioVhEArCIBGARCcciEo5FJByLSDgWkXAsIuFYRMKxiARMzdiXREKgWnbovb9ApHFjeg/nF8nYT9nryfRt6j2e4ybZ583Dw6Uk8Yt1Y5/+K6TBTmXT6fDYRarj9eaqfCTvF4JR8rMFwxqDVF27mSH9SKPQj+bAN7ZX5gIJx5oQqYiVN0t0k+L15fwfx7BwJBxLANKXauWq0+FpqlV/UA1vAFgwEo4lACnV4fDsR7hIfUewcCQcSwwSgoUj4Vhl3eCWgoRg7Rq7kSFdx5FwLDlNArCKwZFwLFFNQrEmR8Kw5CABWFMhAVgikYpYTq/tZUD70yOBWHVjG3KQZjBa2V9tatmPRPr7fN25djmIjX1S0mbxNH59tHyG5JvNJafVZ2lILlIf8u8e5CuN/CncErD6jeTNym+oZHsx1epEFJIOj4fx+sVglNKw8n11/tl/dkLhlTQKByKRiIUjEQtHmhGWVk4YErEAJGJhSDPAio19BZxZMpBKb1a7eysoxGl1XyxSmVi7prcVFJLq8JFopFKwTHcvKGSgK1v5FkA8Eo5FJBwLv3ebXyQcC793k4+EYxEJxyISjkUkHItIKFbbPsuQ+nHbPiASGiLhcVptEAlIqtU7IkGNCveJBCS/D0u1Sog0ERaRcCwi4VhEArFcpDpEKgELQCIWgEQsFIlYOBKxsvePiHQOFo5ErBcjpPdjkJhvtyuXMrSF4D/LT5deJlztCVGzAAAAAElFTkSuQmCC
linkUrl: azavea.com
municipality: Philadelphia
region: PA
```

## Testing Instructions

- `scripts/update`
- Create an organization by using `scripts/manage update-organization <config>.yaml` pointed to a YAML file in the `src/manage` directory
- Go to the organization homepage and verify it looks correct
- Update the organization by updating the YAML file and rerunning `scripts/manage update-organization <config>.yaml` and verify it updates the data displayed on the frontend

Closes #225
